albatross/core/priors.h

```diff
@@ -166,10 +191,14 @@ class LogNormalPrior : public Prior {
 
 } // namespace albatross
 
-CEREAL_REGISTER_TYPE(albatross::UninformativePrior);
-CEREAL_REGISTER_TYPE(albatross::PositivePrior);
-CEREAL_REGISTER_TYPE(albatross::UniformPrior);
-CEREAL_REGISTER_TYPE(albatross::GaussianPrior);
-CEREAL_REGISTER_TYPE(albatross::LogNormalPrior);
+CEREAL_REGISTER_TYPE_WITH_NAME(albatross::UninformativePrior,
```

Comment:
@benjamin0 note that this change should make the serialized JSON files a *little* easier to read, but at the price of non-backward compatible serialization.  Let me know if you think it'll be too much of a pain to recreate the serialized models we're currently using.